.dockerignore

```diff
@@ -1,8 +1,7 @@
+**/*.env*
```

Comment:
env.example should not be ignored

Reply:
That was from the triall branch, will fix

Reply:
fied in next commit

Reply:
fixed in next commit